egs2/wsj0_2mix_spatialized/enh_asr1/conf/tuning/train_enh_asr_beamformer_fbank_transformer.yaml

```diff
@@ -0,0 +1,131 @@
+# minibatch related
+batch_type: folded
+batch_size: 16  # A6000 x 1
+accum_grad: 2
+grad_clip: 5
+max_epoch: 50
+patience: 10
+# The initialization method for model parameters
+init: xavier_uniform
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - acc
+    - max
+-   - train
+    - loss
+    - min
+keep_nbest_models: 10
+num_att_plot: 0
+
+# network architecture
+enh_encoder: stft
+enh_encoder_conf:
+    n_fft: 512
+    hop_length: 128
+    use_builtin_complex: False
+enh_decoder: stft
+enh_decoder_conf:
+    n_fft: 512
+    hop_length: 128
+enh_separator: wpe_beamformer
+enh_separator_conf:
+    num_spk: 2
+    loss_type: spectrum
+    use_wpe: False
+    wnet_type: blstmp
+    wlayers: 3
+    wunits: 300
+    wprojs: 320
+    wdropout_rate: 0.0
+    taps: 5
+    delay: 3
+    use_dnn_mask_for_wpe: True
+    use_beamformer: True
+    bnet_type: blstmp
+    blayers: 3
+    bunits: 512
+    bprojs: 512
+    badim: 320
+    ref_channel: 0
+    use_noise_mask: False
+    beamformer_type: mvdr
+    bdropout_rate: 0.0
+    shared_power: False
+enh_criterions:
+  # The first criterion
+  - name: si_snr
+    conf:
+        eps: 1.0e-7
+    # the wrapper for the current criterion
+    # for single-talker case, we simplely use fixed_order wrapper
+    wrapper: fixed_order
```

Comment:
This should be PIT, right? We should also explicitly indicate whether the permutation is determined by enh_loss or ctc_loss. Ignore this comment if there is already such indication.

Reply:
You are right. I will handle this issue.